python/cudf/cudf/core/column/column.py

```diff
@@ -227,15 +228,23 @@ def _concat(cls, objs, dtype=None):
             if not (obj.dtype == head.dtype):
                 raise ValueError("All series must be of same type")
 
+        newsize = sum(map(len, objs))
+        if newsize > libcudfxx.MAX_COLUMN_SIZE:
+            raise MemoryError("Result of concat cannot have size > INT32_MAX")
```

Comment:
magic string "INT32_MAX" will be wrong when `libcudfxx.MAX_COLUMN_SIZE` changes.